docs/README.md

```diff
@@ -0,0 +1,44 @@
+# Docs
+Meta information about this project's documentation.
+
+Table of contents:
+- [Building the docs](#Building)
+- [Writing docstrings](#Docstrings)
+- [Writing a changelog](#Changelog)
+
+
+## Building
+To build the docs, you can use the following task:
+```shell
+poetry run task docs
+```
+
+The output will be in the [`/docs/build`](.) directory.
+
+Additionally, there are two helper tasks: `apidoc` and `builddoc`.
+`apidoc` is responsible for calling autodoc and generating docs from docstrings.
+`builddoc` generates the HTML site, and should be called after apidoc.
```

Comment:
```suggestion
Additionally, there are two helper tasks: `apidoc` and `builddoc`:

- `apidoc` is responsible for calling autodoc and generating docs from docstrings.
- `builddoc` generates the HTML site, and should be called after apidoc.
```

Reply:
This seems very nit picky, and I actually prefer it as one paragraph.